src/main/java/core/basesyntax/service/ParserInFruitTransaction.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ParserInFruitTransaction {
```

Comment:
```suggestion
public interface ParserService {
```
I'd recommend naming it this way to make it core clear and concise